staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -238,7 +238,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	AdvancedAuditing: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.28
 
-	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Alpha},
+	ValidatingAdmissionPolicy: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
if enabling this feature gate by default will make the admission plugin unconditionally try to watch a disabled-by-default beta API, that's not great.

we should do one of these:
* leave the gate disabled and only enable it by default when we enable the API (at GA)
* add an additional check before starting the informers/controllers to make sure the feature gate is enabled AND we are serving the API